lib/controllers/images_to_pdf.dart

```diff
@@ -27,4 +28,7 @@ class ImageToPdfController extends GetxController {
   }
 
   void removeImage(int index) => _images.removeAt(index);
+  void toastMsg(String msg) {
+    Fluttertoast.showToast(msg: msg, gravity: ToastGravity.BOTTOM);
```

Comment:
i mean use toast/snackbar from get package not the new ```fluttertoast``` if you have question regarding it leave here